src/js/extensions/auto-link.js

```diff
@@ -41,9 +41,20 @@
             this.document.execCommand('AutoUrlDetect', false, false);
         },
 
+        isLastInstance: function () {
+            var activeInstances = 0;
+            for (var i = 0; i < window._mediumEditors.length; i++) {
+                var editor = window._mediumEditors[i];
```

Comment:
Each extension has a reference to `this.window` and `this.document`, which are equivalent to the `contentWindow` and `ownerDocument` options that can be optionally passed into the editor.  Could you update these references to `window` here to be `this.window` instead?